x-pack/plugins/reporting/server/routes/common/jobs/get_job_routes.ts

```diff
@@ -109,6 +117,12 @@ export const commonJobsRouteHandlerFactory = (reporting: ReportingCore) => {
 
         await jobsQuery.delete(docIndex, docId);
 
+        // event tracking of the deleted report
+        const eventTracker = reporting.getEventTracker(docId, doc.jobtype, doc.payload.objectType);
+        const timeSinceCreation =
+          new Date(Date.now()).valueOf() - new Date(doc.created_at).valueOf();
```

Comment:
```suggestion
          new Date().valueOf() - new Date(doc.created_at).valueOf();
```

Reply:
Maybe, even more concise:

```suggestion
          Date.now() - new Date(doc.created_at).valueOf();
```